integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -36,6 +36,8 @@ const testRunWorkflowOnSnapshotFn = _.flow(
   await click(page, clickable({ textContains: 'Start with an existing workspace' }))
   await select(page, 'Select a workspace', workspaceName)
   await click(page, clickable({ text: 'Import' }))
+  // Wait for bucket access to avoid sporadic failure when launching workflow.
+  await checkBucketAccess(page, billingProject, workspaceName)
```

Comment:
Unfortunately this likely will not fix all instances of bucket access not being available, as I posted in [Slack](https://broadinstitute.slack.com/archives/C01EHNUM73R/p1644354756201859).

Reply:
I do think it is worth adding this check to address the more common problem of access not being granted fast enough for it to be present at the time that the workflow is launched. The fact that access can fail after having previously succeeded is a different (and very disappointing) issue.